scripts/scil_recognize_single_bundle.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+    Compute a simple Recobundles (single-atlas & single-parameters).
+    The model need to be cleaned and lightweight.
+    Transform should come from ANTs: (using the --inverse flag)
+    AntsRegistration -m MODEL_REF -f SUBJ_REF
+    ConvertTransformFile 3 0GenericAffine.mat 0GenericAffine.npy --ras --hm
+"""
+
+import argparse
+import pickle
+
+from dipy.segment.bundles import RecoBundles
+from dipy.tracking.streamline import transform_streamlines
+import nibabel as nib
+from nibabel.streamlines.array_sequence import ArraySequence
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exists)
+
+
+def buildArgsParser():
+    p = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter,
+        description=__doc__,
+        epilog="Garyfallidis, E., Côté, M. A., Rheault, F., Sidhu, J., Hau, J.,"
+        "Petit, L., ... & Descoteaux, M. (2018). Recognition of white matter"
```

Comment:
You just remove Fortin and Cunnane. You should either shorten it up more, or keep everyone.

Also, please add some line breaks. It doesn't look good when compared to the rest of the help message.